content/en/dynamic_instrumentation/_index.md

```diff
@@ -29,6 +29,12 @@ further_reading:
   text: "Use Datadog Dynamic Instrumentation to add application logs without redeploying"
 ---
 
+{{% site-region region="gov" %}}
+<div class="alert alert-warning">
+  Dynamic Instrumentation is not supported on US1-FED, as it requires <a href="/agent/remote_config/">Remote Configuration</a> to be enabled.
```

Comment:
```suggestion
 Dynamic Instrumentation is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}) as it requires <a href="/agent/remote_config/">Remote Configuration</a> to be enabled.
```

I saw someone use this format the other day which might be better than using the literal `US1-FED`. Just a suggestion though!